modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc

```diff
@@ -0,0 +1,93 @@
+/ Module included in the following assemblies:
+//
+// * observability/otel/otel-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="otel-troubleshooting-instrumentation-injection-into-your-workload_{context}"]
+= Troubleshooting instrumentation injection into your workload
+
+To troubleshoot instrumentation injection, you can perform the following activities:
+
+* Checking if the `Instrumentation` object was created
+* Checking if the init-container started
+* Checking if the resources were deployed in the correct order
+* Searching for errors in the Operator logs
+* Double-checking the pod annotations
+
+.Procedure
+
+. Run the following command to verify that the `Instrumentation` object was successfully created:
++
+[source,console]
+----
+oc get instrumentation -n <workload_project> # <1>
+----
+<1> The namespace where the instrumentation was created.
+
+. Run the following command to verify that the `opentelemetry-auto-instrumentation` init-container successfully started, which is a prerequisite for instrumentation injection into workloads:
++
+[source,console]
+----
+oc get events -n <workload_project> # <1>
+----
+<1> The namespace where the instrumentation is injected for workloads.
++
+.Example output
+[source,console]
+----
+... Created container opentelemetry-auto-instrumentation
+... Started container opentelemetry-auto-instrumentation
+----
+
+. Verify that the resources were deployed in the correct order for the auto-instrumentation to work correctly. The correct order is to deploy the `Instrumentation` custom resource (CR) before the application. For information about the `Instrumentation` CR, see the section "Configuring the instrumentation".
```

Comment:
Drop "the section":
```suggestion
. Verify that the resources were deployed in the correct order for the auto-instrumentation to work correctly. The correct order is to deploy the `Instrumentation` custom resource (CR) before the application. For information about the `Instrumentation` CR, see "Configuring the instrumentation".
```
And see again Janelle's earlier comment via https://github.com/openshift/openshift-docs/pull/82280/files#r1837152975 for more.